tutorials/fundamentals/installation.md

```diff
@@ -1,30 +1,47 @@
-# napari installation tutorial
+# How to install napari on your machine
 
-Welcome to the **napari** installation tutorial!
+Welcome to the **napari** installation guide!
 
-This tutorial will teach you how to do a clean install of **napari**. It is
-aimed at people that just want to use napari. For people also interested in
-contributing to napari please check our [contributing
-guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
-for more advanced installation procedures. At the end of the tutorial you should
-have napari successfully installed on your computer and be able to make the
-napari viewer appear.
+This guide will teach you how to do a clean install of **napari** and launch the viewer.
+
+## Prerequisites
+
+Prerequisites differ depending on what type of distribution you want to install.
+
+### Installing the bundled app
+This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
+This installation method does not have any prerequisites. 
+[Click here](#installing-as-a-bundled-app) to see instructions
+for installing the bundled app.
 
-## installation
+### Installing as a python package or in editable mode
+This installation method allows you to use napari from Python to programmatically 
+interact with the app.
 
-### which distribution to install
+It requires:
+- [Python 3.7+](https://www.python.org/downloads/)
+- the ability to install python packages via [pip](https://pypi.org/project/pip/) OR [conda-forge](https://conda-forge.org/docs/user/introduction.html)
+
+You may also want:
+- an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
+[venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) (For editable mode)
+
+***
+
+### Choose which distribution to install.
```

Comment:
```suggestion
## Choose which distribution to install.
```
Going up one level ensures people understand the next sections are options and they should only go with one of them. I'd go as far as using numbering to make that even clearer:

1. From pip
2. From conda-forge
3. From the main branch on GitHub

Reply:
Yeah I think it would be nice if this was a dropdown selection similar to [this section](https://numpy.org/numpy-tutorials/content/pairing.html#pair-your-notebook-files-ipynb-and-md) in the numpy notebook/jupytext guide:

1. It would limit the amount of code those who want to install the bundled app see at first glance if scrolling through the document
2. It would limit distractions for those who just want to install via pip or via conda - you probably don't want to do all of them and if you do, it doesn't feel like that much effort to click all three
3. It clearly shows that these are three separate options and in general you would pick just one.

Thoughts? I'm not sure how easy this would be to implement given the current theme because I don't remember seeing any drop down/accordion behaviour. Pinging @codemonkey800 - do we have something like this? If not, could we bring over something like the [FAQ dropdown](https://www.napari-hub.org/faq) from the hub?

Reply:
That is a [jupyter-book feature](https://jupyterbook.org/interactive/hiding.html) so might be already doable - although I don't know about the styling using the napari theme.

Reply:
The napari tutorials uses this feature extensively, so we'll also have support for it in the napari theme 😄 

You can see a demo of it in #179.